vaccine_feed_ingest/runners/mn/gov/normalize.py

```diff
@@ -0,0 +1,209 @@
+    # accidental mutation.
+    site_name = "gov"
+    runner = "mn"
+
+    # Could parse these from the input file name, but do not for now to avoid
+    # accidental mutation.
+    arcgis = "18279cf4ab4740ae92708db437250056"
+    layer = 0
+
+    return f"{runner}_{site_name}:{arcgis}_{layer}_{data_id}"
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = []
+
+    if phone := site["attributes"]["Phone"]:
+        contacts.append(schema.Contact(phone=phone))
+
+    def cleanup_url(url):
+        if not url:
+            return None
+        if "@" in url:
+            # Some of these are email addresses.
+            # Skipping those for now.
+            return None
+        if url == "COVID-19 Vaccine Information - Hennepin Healthcare":
+            return None
+        if url == "https/:www.Lewisdrug.com":
+            url = "https://www.lewisdrug.com"
+
+        if not url.startswith("http"):
+            url = "http://" + url
+
+        return url
+
+    if url := cleanup_url(site["attributes"]["URL"]):
+        contacts.append(schema.Contact(website=url))
+
+    if contacts:
+        return contacts
+
+    return None
+
+
+def _get_notes(site: dict) -> Optional[List[str]]:
+    notes = []
+
+    if hours_info := site["attributes"]["AdditionalHoursInfo"]:
+        notes.append(hours_info)
+
+    if requirements := site["attributes"]["TestingRequirements"]:
+        maybe_notes = requirements.split("|")[-1]
+        if maybe_notes.count(" ") > 3:
+            notes.append(maybe_notes)
+
+    if notes:
+        return notes
+
+    return None
+
+
+def _get_opening_dates(site: dict) -> Optional[List[schema.OpenDate]]:
+    start_date = site["attributes"]["StartDate"]
+    end_date = site["attributes"]["EndDate"]
+
+    if start_date or end_date:
+        return [schema.OpenDate(opens=start_date, closes=end_date)]
+
+    return None
+
+
+def _get_opening_hours(site: dict) -> Optional[List[schema.OpenHour]]:
+    open_hours = []
+
+    def _parse_time(t: str) -> datetime.time:
+        # example: "8:30am"
+        hh, mmxx = t.split(":")
+        if hh == "12":
+            hh = "0"
+        return datetime.time(int(hh) + (12 if mmxx[2] == "p" else 0), int(mmxx[:2]))
+
+    for day in [
+        "Sunday",
+        "Monday",
+        "Tuesday",
+        "Wednesday",
+        "Thursday",
+        "Friday",
+        "Saturday",
+    ]:
+        start = site["attributes"][f"{day}Start"]
+        end = site["attributes"][f"{day}End"]
+        if start and end and (start != "Closed") and (end != "Closed"):
+            try:
+                open_hours.append(
+                    schema.OpenHour(
+                        day=day.lower(),
+                        opens=_parse_time(start),
+                        closes=_parse_time(end),
+                    )
+                )
+            except ValidationError:
+                pass  # ignore hours that were entered incorrectly
```

Comment:
Can you add a log statement for these validation errors so we can see what they are?